validation/presos.tsx

```diff
@@ -0,0 +1,8 @@
+import { object, string } from "yup"
+
+export const validatePresos = object({
+    url: string().url().required(),
+    eventName: string().required(),
+    title: string(),
+    eventLocation:string()
+})
```

Comment:
It would be better if the validation is co-located with the form because they're related. Having this so far away from its use adds additional complexity for no benefit.